manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -355,7 +355,7 @@ spec:
               name: vsphere-config-volume
               readOnly: true
         - name: csi-provisioner
-          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.2.1
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.3.0
```

Comment:
Here also [v3.3.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.3.0) have a few feature gates turned on by default. have we analyzed if there's any impact to the driver?

Reply:
There is no impact on the driver as such.